src/components/NamespaceFilter/a

```diff
@@ -0,0 +1,34 @@
+{currentFilterType: {…}, filterTypeList: Array(4), activeFilters: Array(0), currentValue: ""}
```

Comment:
I don't understand if this file is an error, or if you want to extract some filtering to a different file.
In any case this should be reviewed.

Reply:
It's an error ^^